README.md

```diff
@@ -14,6 +14,7 @@
 
 A Yarn Workspace containing packages to collectively form the SDK, written in TypeScript.
 
+- [@cardano-sdk/cardano-graphql-provider](packages/cardano-graphql-provider)
```

Comment:
These links are broken when on the docs homepage, I'm not sure how to fix?

Reply:
I've fixed this by disabling the injection of the readme, and just have the module index as the landing page. We don't need this to be a fully self-contained site, as the codebase links to these docs

```
readme": "none"
```